client/src/client.rs

```diff
@@ -649,6 +648,7 @@ impl Client {
         &mut self,
         event_filter: FilterBox,
     ) -> Result<impl Iterator<Item = Result<Event>>> {
+        iroha_logger::trace!(?event_filter);
```

Comment:
I think, rewriting our `log` macro to do something like that will be easy, so that we shouldn't write it by hand

Reply:
Fine as is for now. You're not saving a line of code either, since all I do is list the arguments that need to be traced (there aren't that many, and you can't do this automatically, since large objects take up valuable CPU time).